DrawerKit/DrawerKit/Internal API/PresentationController+Gestures.swift

```diff
@@ -23,23 +23,15 @@ extension PresentationController {
 
         switch panGesture.state {
         case .began:
-            lastDrawerState = GeometryEvaluator.drawerState(for: currentDrawerY,
-                                                            drawerPartialHeight: drawerPartialHeight,
-                                                            containerViewHeight: containerViewHeight,
-                                                            configuration: configuration,
-                                                            clampToNearest: true)
+            break
```

Comment:
I don't understand why you're no longer setting `lastDrawerState` here.

Reply:
It is not necessary to update it during the course of the pan.

Reply:
`lastDrawerState` is not really consumed by anything until this PR, by the way.

Reply:
I think it might be, for the benefit of along-side animations performed by the presenting and/or presented view controllers.

Reply:
After looking into this, it is actually setting the `targetDrawerState` (renamed from `lastDrawerState`).